### ISSUE TITLE
Cannot change authentication to "Forms"

### ISSUE THREAD

**Issue by brunobraga** [Radarr/Radarr#6913]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

From a clean install of Radarr [Radarr.master.3.2.2.5080.windows-core-x64-installer.exe], if I change the authentication from None to Forms and place a user and password, upon restart, it throws the following error on the http://localhost:7878/ path:

`
{
  "message": "The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.",
  "description": "System.FormatException: The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.\r\n   at System.Convert.FromBase64CharPtr(Char* inputPtr, Int32 inputLength)\r\n   at System.Convert.FromBase64String(String s)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.DecryptAndValidateAuthenticationCookie(String cookieValue, FormsAuthenticationConfiguration configuration)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.GetAuthenticatedUserFromCookie(NancyContext context, FormsAuthenticationConfiguration configuration)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.\u003C\u003Ec__DisplayClass11_0.\u003CGetLoadAuthenticationHook\u003Eb__0(NancyContext context)\r\n   at Nancy.BeforePipeline.\u003C\u003Ec__DisplayClass8_0.\u003CWrap\u003Eb__0(NancyContext ctx, CancellationToken ct)\r\n   at Nancy.BeforePipeline.Invoke(NancyContext context, CancellationToken cancellationToken)\r\n   at Nancy.NancyEngine.InvokeRequestLifeCycle(NancyContext context, CancellationToken cancellationToken, IPipelines pipelines)"
}
`

### Expected Behavior

I was expecting to see the login screen.

### Steps To Reproduce

1 - start from a clean Windows 10 64-bit install and Radarr v3.2.2.5080
2 - go to Settings > General > Security > Authentication
3 - change from None to Forms
4 - place a user and password
5 - save changes and restart
6 - reopen the radarr  

### Environment

```markdown
- OS: Windows 10 64-bit
- Radarr: 3.2.2.5080
- Docker Install: no
- Using Reverse Proxy: no
- Browser: Edge
```

### What branch are you running?

Master

### Trace Logs?

The trace pops up on the main screen, as a json format:

`
{
  "message": "The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.",
  "description": "System.FormatException: The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.\r\n   at System.Convert.FromBase64CharPtr(Char* inputPtr, Int32 inputLength)\r\n   at System.Convert.FromBase64String(String s)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.DecryptAndValidateAuthenticationCookie(String cookieValue, FormsAuthenticationConfiguration configuration)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.GetAuthenticatedUserFromCookie(NancyContext context, FormsAuthenticationConfiguration configuration)\r\n   at Nancy.Authentication.Forms.FormsAuthentication.\u003C\u003Ec__DisplayClass11_0.\u003CGetLoadAuthenticationHook\u003Eb__0(NancyContext context)\r\n   at Nancy.BeforePipeline.\u003C\u003Ec__DisplayClass8_0.\u003CWrap\u003Eb__0(NancyContext ctx, CancellationToken ct)\r\n   at Nancy.BeforePipeline.Invoke(NancyContext context, CancellationToken cancellationToken)\r\n   at Nancy.NancyEngine.InvokeRequestLifeCycle(NancyContext context, CancellationToken cancellationToken, IPipelines pipelines)"
}
`

[AB#2327](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2327)

**Comment by bakerboy448** [comment]
what non A-Z0-9 characters are you using for your username and password?

**Comment by ta264** [comment]
If this is an issue I think it's to do with Nancy which we have dropped in v4. So I'm going to close this and if you can replicate it on v4 then we'll reopen and investigate. We won't fix it on v3.  As bakerboy suggests, I think there's something odd with your password, plenty of people use this functionality.

**Comment by antoineguilbert** [comment]
I've the same issue. 
Radarr deployed from docker compose with this image : lscr.io/linuxserver/sonarr. 

When I active the login form, it works with the local IP address but trough the reverse proxy, I've got this error too. 

"The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters."

I thought first it's my password but I've tried with "admin" as user and "admin" as password. Same problem. 
Any idea ?
